packages/playwright-test/src/runner.ts

```diff
@@ -381,6 +383,8 @@ export class Runner {
     if (fatalErrors.length) {
       for (const error of fatalErrors)
         this._reporter.onError?.(error);
+      if (options.passWithNoTests && noTests && fatalErrors.length === 1)
```

Comment:
I would rather not add the error in the first place.

Reply:
Done.